website/source/docs/faq.html.markdown

```diff
@@ -20,18 +20,18 @@ and [`disable_update_check`](/docs/agent/options.html#disable_update_check).
 
 ## Q: How does Atlas integration work?
 
-Consul makes use of a HashiCorp service called [SCADA](http://scada.hashicorp.com)
-(Supervisory Control And Data Acquisition). The SCADA system allows clients to maintain
-long-running connections to Atlas. Atlas can in turn provide auto-join facilities for
-Consul agents (supervisory control) and an integrated dashboard showing the health of
-all connected agents (data acquisition).
+Hosted Consul Enterprise in Atlas was officially deprecated on March 7th,
+2017.
 
-Standard ACLs can be applied to the SCADA connection, ensuring that Atlas is given only
-those privileges that make sense for your deployment.
+There are strong alternatives available and they are listed below.
 
-Using the SCADA service is optional. SCADA is only enabled by opt-in.
+For users on AWS the [-retry-join-ec2 configuration options](https://www.consul.io/docs/agent/options.html#_retry_join_ec2_tag_key) allow bootstrapping by automatically discovering AWS instances with a given tag key/value at startup. For users not on AWS the native [-join and retry-join functionality](https://www.consul.io/docs/agent/options.html#_join) can be used.
```

Comment:
Can you add pointers to the GCE stuff as well? These links can also get the `https://www.consul.io` stripped off the front.